instructure/canvas-ios
# Offline sync selection persistency

refs: MBL-16759
affects: Student
release note: none

test plan:
- Open offline sync selection screen.
- Select a few items.
- Close the screen.
- Open it again.
- Selections should be restored.
- Play around with whole course and files tab selection.
- Selections should be restored after app restart.
- Opening the selection screen from the dashboard or from a course should display consistent selections.

## Checklist
